packages/types/src/hex.ts

```diff
@@ -1,4 +1,3 @@
-import { Buffer } from 'buffer';
```

Comment:
We don't want to rely upon globals

Reply:
My overall feedback is that we need to find a better solution in general to this problem. Moving to a reliance upon globals in our production app in order to run our tests is not a good approach.